engine/gamesys/src/gamesys/test/test_gamesys.h

```diff
@@ -39,8 +38,38 @@ struct Params
 };
 
 template<typename T>
-class GamesysTest : public jc_test_params_class<T>
+class CustomizableGamesysTest : public jc_test_params_class<T>
+{
+public:
+    struct ProjectOptions {
+      uint32_t m_MaxCollisionCount;
+      uint32_t m_MaxContactPointCount;
+      bool m_3D;
+    };
+
+    CustomizableGamesysTest() {
+      memset(&projectOptions, 0, sizeof(projectOptions));
+    }
+
+    ProjectOptions projectOptions;
```

Comment:
We use the "m_" for members.
"projectOptions" -> "m_PropjectOptions"

Reply:
fixed